### PR TITLE
[audioplayers] Add integration test

### PR DESCRIPTION
**Pull request by XuWanchaoS**
- Add integration test
- Update CHANGELOG.md and README.md
- Fix the timer doesn't work on tizen tv.

### CODE REVIEW

File: packages/audioplayers/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 ## 1.0.0
 
 * Initial release.
+
+## 1.0.1
+
+* Add integration_test
```
Comment (HakkyuKim) [inline packages/audioplayers/CHANGELOG.md:7]:
```suggestion
* Add integration test.
```

File: packages/audioplayers/example/pubspec.yaml [hunk 2]
```diff
@@ -13,9 +13,19 @@ dependencies:
...
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_driver:
+    sdk: flutter
+  integration_test:
+    sdk: flutter
+  integration_test_tizen:
+    path: ../../integration_test/
+  pedantic: ^1.10.0
```
Comment (HakkyuKim) [inline packages/audioplayers/example/pubspec.yaml:25]: `pedantic` is [deprecated](https://pub.dev/packages/pedantic). Packages will use the `analysis_options.yaml` in the repo root, you can remove the dependency.

**Review by HakkyuKim — APPROVED** [review]
LGTM with minor suggestions.

**Comment by XuWanchaoS** [comment]
I have fixed source as review result.

File: packages/audioplayers/example/pubspec.yaml [hunk 3]
```diff
@@ -13,9 +13,18 @@ dependencies:
   path_provider_tizen: ^2.0.0
   provider: ^5.0.0
 
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_driver:
```
Comment (bbrto21) [inline packages/audioplayers/example/pubspec.yaml:19]: minor thing : sort pub dependencies
https://dart-lang.github.io/linter/lints/sort_pub_dependencies.html

**Review by bbrto21 — APPROVED** [review]

**Pull request merged**